src/modExcelToXML.bas

```diff
@@ -0,0 +1,169 @@
+Attribute VB_Name = "modExcelToXML"
+Option Explicit
+
+Const ms As Double = 0.000000011574
+
+Sub excelToXML()
+    
+    If Not changeFileExtension(ThisWorkbook.Path, ThisWorkbook.Name) Then
+            
+    End If
+        
+    If Not windowsUnZip(Left(ThisWorkbook.FullName, Len(ThisWorkbook.FullName) - 4) & "zip", ThisWorkbook.Path) Then
+    
+    End If
+        
+End Sub
+
+Function changeFileExtension(strPath As String, strFileName As String) As Boolean
+    
+    Dim FSO As New FileSystemObject
```

Comment:
I have declared FSO as a global public variable in modUtil. I am starting to think that is not a good way to do things and this way is better. Do you think I should remove the global public in modUtil and use this method throughout the rest of the code?